src/stores/Create.ts

```diff
@@ -42,22 +42,37 @@ export interface FormState {
 }
 
 export enum FormStatus {
+    INIT = 'idle',
```

Comment:
The FormStatus is really more like a cheap and crappy state machine at this point. Idle is no longer useful so renaming to `init` to represent that "nothing happened yet".